features/storage/TESTS/blockdevice/heap_block_device/main.cpp

```diff
@@ -28,6 +28,10 @@ using namespace utest::v1;
 #define TEST_BLOCK_COUNT 10
 #define TEST_ERROR_MASK 16
 
+#if ((MBED_RAM_SIZE - MBED_BOOT_STACK_SIZE) <= TEST_BLOCK_DEVICE_SIZE)
```

Comment:
Why is this condition different than the other two files but the comment the same?

Reply:
Because the `heap_block_device` test case allocates memory using TEST_BLOCK_DEVICE_SIZE.
https://github.com/ARMmbed/mbed-os/blob/a707fd133e6230d8e5e35971d7edffc2b12b66dd/features/storage/TESTS/blockdevice/heap_block_device/main.cpp#L45

And other test cases use different symbols.
https://github.com/ARMmbed/mbed-os/blob/a707fd133e6230d8e5e35971d7edffc2b12b66dd/features/storage/TESTS/blockdevice/mbr_block_device/main.cpp#L35